Java/src/main/java/epam/Planes/Plane.java

```diff
@@ -0,0 +1,55 @@
+package epam.Planes;
+
+import java.util.Objects;
+
+
+abstract public class Plane {
+	String model;
```

Comment:
please add access modifier, and if it is necessary, please create the getMethod

Reply:
Private access modifier added, also get method created